### PR TITLE
removed the commented dead code and the log statement.

### PR DESCRIPTION
**Pull request by talhainvenxion**
In this PR, we removed commented-out dead code and console.log statements. 
@Cahllagerfeld, I understand that this PR contains many changes, but these modifications won't impact functionality.

### CODE REVIEW

**Comment by htahir1** [comment]
@talhainvenxion Do you also see the deepscan report (https://deepscan.io/dashboard/#view=project&tid=21882&pid=25246&bid=787377&subview=pull-request&prid=1697353) when you make a PR? it shows you code quality issues this PR has introduced... maybe fix these as well?

**Comment by talhainvenxion** [comment]
> @talhainvenxion Do you also see the deepscan report (https://deepscan.io/dashboard/#view=project&tid=21882&pid=25246&bid=787377&subview=pull-request&prid=1697353) when you make a PR? it shows you code quality issues this PR has introduced... maybe fix these as well?

Yes, I have fixed them.

File: src/ui/layouts/connectors/Connectors/List/useService.ts [hunk 1]
```diff
@@ -3,20 +3,13 @@
...
 import {
-  // workspaceSelectors,
-  // secretPagesSelectors,
-  secretPagesSelectors,
-  // secretSelectors,
-  // secretSelectors,
   connectorSelectors,
+  secretPagesSelectors,
 } from '../../../../../redux/selectors';
-import { getFilteredDataForTable } from '../../../../../utils/tableFilters';
```
Comment (Cahllagerfeld) [inline src/ui/layouts/connectors/Connectors/List/useService.ts:19]: some imports are getting deleted, these were unused I assume

**Review by Cahllagerfeld — COMMENTED** [review]
Besides that question its looking good to me. Some rough smoke tests on the deploy preview also didn't break

**Comment by Cahllagerfeld** [comment]
I'm a bit confused about the CLA Bot, giving a note. Can you please try to resign the CLA, so it removes the warning 🙏

**Comment by talhainvenxion** [comment]
> I'm a bit confused about the CLA Bot, giving a note as you contributed heavily before. Can you please try to resign the CLA, so it removes the warning 🙏

I signed it but it is still pending.

**Pull request merged**